<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>41</width>
      <height>9</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>104</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="text">
     <string>editer</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>display1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>display2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>clean</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FOR smart Next</family>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionchoose_your_signal_table"/>
    <addaction name="actionchoose_your_svg_file"/>
    <addaction name="actionchoose_your_svg_folder"/>
    <addaction name="actionchosse_your_gcid_wdid_table"/>
    <addaction name="actionchoose_your_CF_table"/>
    <addaction name="actionexit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuversion">
    <property name="title">
     <string>Version</string>
    </property>
    <addaction name="action1_0"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuversion"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionchoose_your_signal_table">
   <property name="text">
    <string>choose your signal table</string>
   </property>
  </action>
  <action name="actionchoose_your_svg_file">
   <property name="text">
    <string>choose your svg file</string>
   </property>
  </action>
  <action name="actionchoose_your_svg_folder">
   <property name="text">
    <string>choose your svg folder</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionchosse_your_gcid_wdid_table">
   <property name="text">
    <string>chosse your gcid-wdid table</string>
   </property>
  </action>
  <action name="actionchoose_your_CF_table">
   <property name="text">
    <string>choose your CF table</string>
   </property>
  </action>
  <action name="action1_0">
   <property name="text">
    <string>1.0</string>
   </property>
  </action>
  <action name="actionFager_system">
   <property name="text">
    <string>Fager system</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
